schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+	id                INT GENERATED BY DEFAULT AS IDENTITY,
```

Comment:
- [ ] <b>OPTIONAL:</b>   It would be nice to add `NOT NULL` and `PRIMARY KEY` constraints on ID just to make sure that it can't accept null values and it is primary key.

Reply:
Thank you for your suggestion.